bots/mlbridge/src/main/java/org/openjdk/skara/bots/mlbridge/ArchiveMessages.java

```diff
@@ -325,7 +325,9 @@ static String composeConversationFooter(PullRequest pr, URI issueProject, String
                 "Commit messages:\n" +
                 formatCommitMessagesBrief(commits, commitsLink).orElse("") + "\n\n" +
                 "Changes: " + pr.changeUrl() + "\n" +
-                (webrev.uri() == null ? "" : " Webrev: " + webrev.uri().toString() + "\n") +
+                (webrev.diffTooLarge() ?
+                        "  Webrev: Webrev is not available because diff is too large\n" :
```

Comment:
This string is repeated a lot, can we make it a constant, at least within this class?

Reply:
Sure, will fix it